docs/how-to/manage-relationships

```diff
@@ -0,0 +1,78 @@
+`Relationship` is the word used by Cozy-Client, to refer relation / association between documents. For that, we need to define our schema.
+
+Ex: 
+```jsx
+const schema = {
+    todos: {
+       doctype: 'io.cozy.todos',
+       relationships: {
+         tasks: {
+           doctype: 'io.cozy.tasks',
+           type: 'has-many'
+         }
+       }
+     }
+  }
+```
+
+Here, it's say that our `todos` document can be associated to `has-many` `tasks` document.
+
+#How to access todos with their associated tasks ? 
```

Comment:
I think we should keep the headers general for the user to quickly find the information (s)he is interested in. "How to fetch documents with their associated relationships"